presto-spi/src/main/java/com/facebook/presto/spi/block/RunLengthEncodedBlock.java

```diff
@@ -56,7 +56,7 @@ public RunLengthEncodedBlock(Block value, int positionCount)
             throw new IllegalArgumentException("positionCount is negative");
         }
 
-        this.value = value;
```

Comment:
looks like syntax typo

(`Add position checks in RunLenghtEncodedBlock` commit)